src/printer.rs

```diff
@@ -263,31 +286,57 @@ impl<W: WriteColor> Printer<W> {
             self.write(b":");
         }
         if self.replace.is_some() {
-            let line = re.replace_all(
-                &buf[start..end], &**self.replace.as_ref().unwrap());
+            let mut count = 0;
+            let line = {
+                let replacer = CountingReplacer { count: &mut count, replace : &**self.replace.as_ref().unwrap() };
```

Comment:
I think we should write code that is consistent with the style of the surrounding code. Could you please adhere to a 79 column limit (inclusive)? Thanks.